tutorials/applications/QAA to solve a MWIS problem.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>I think strictly speaking, without the second constraint there is nothing enforcing the independent set to have the maximum number of nodes. I would simplify this to</p><blockquote>If all the weights of the nodes are the same, then this problem is called the Maximally Independent Set (MIS).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>